code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -390,3 +390,95 @@
 		whistle.whistler = null
 		whistle = null
 	return ..()
+
+/obj/item/runic_vendor_staff
+	name = "staff of runic vendormancy"
+	desc = "This staff allows you to summon, detonate and throw Runic Vendors. A modern spin on the old Geomancy with 3 summoning charges that can be recovered with a simple magical channeling."
+	icon_state = "vendor_staff"
+	inhand_icon_state = "vendor_staff"
+	lefthand_file = 'icons/mob/inhands/weapons/staves_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/staves_righthand.dmi'
+	icon = 'icons/obj/weapons/guns/magic.dmi'
+	slot_flags = ITEM_SLOT_BACK
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 18
+	damtype = BURN
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	attack_verb_continuous = list("sears", "clubs", "burn")
+	attack_verb_simple = list("sear", "club", "burn")
+	hitsound = 'sound/weapons/sear.ogg'
+
+	/// Range cap on where you can summon vendors.
+	var/max_summon_range = 7
+	/// Channeling time to summon a vendor.
+	var/summoning_time = 1 SECONDS
+	/// Checks if the staff is channeling a vendor already.
+	var/staff_is_busy_summoning = FALSE
+	///Number of summoning charges left.
+	var/summon_vendor_charges = 3
+
+/obj/item/runic_vendor_staff/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!check_allowed_items(target, not_inside = TRUE))
+		return
+	. |= AFTERATTACK_PROCESSED_ITEM
+	var/turf/T = get_turf(target)
+	if(istype(target, /obj/machinery/vending/runic_vendor))
+		var/obj/machinery/vending/runic_vendor/runic_explosion_target = target
+		runic_explosion_target.runic_explosion()
+		return
+	var/obj/machinery/vending/runic_vendor/vendor_on_turf = locate() in T
+	if(vendor_on_turf)
+		vendor_on_turf.runic_explosion()
+		return
+	if(!summon_vendor_charges)
+		user.balloon_alert(user, "no charges!")
+		return
+	if(get_dist(T,src) > max_summon_range)
+		user.balloon_alert(user, "too far!")
+		return
+	if(get_turf(src) == T)
+		user.balloon_alert(user, "too close!")
+		return
+	if(staff_is_busy_summoning)
+		user.balloon_alert(user, "already summoning!")
+		return
+	if(T.is_blocked_turf(TRUE))
+		user.balloon_alert(user, "blocked!")
+		return
+	if(summoning_time)
+		staff_is_busy_summoning = TRUE
+		user.balloon_alert(user, "summoning...")
+		if(!do_after(user, summoning_time, target = target))
+			staff_is_busy_summoning = FALSE
+			return
+		staff_is_busy_summoning = FALSE
+	if(summon_vendor_charges)
+		playsound(src,'sound/weapons/resonator_fire.ogg',50,TRUE)
+		user.visible_message(span_warning("[user] summons a runic vendor!"))
+		new /obj/machinery/vending/runic_vendor(T)
+		summon_vendor_charges--
+		user.changeNext_move(CLICK_CD_MELEE)
+
+/obj/item/runic_vendor_staff/attack_self(mob/user, modifiers)
+	. = ..()
+	user.balloon_alert(user, "recharging...")
+	if(!do_after(user, 5 SECONDS))
+		return
+	user.balloon_alert(user, "fully charged")
+	summon_vendor_charges = 3
```

Comment:
No interaction with the charge spell? A little lame

Reply:
I forgot that it existed, the charge spell recharges the Scepter now.